### PR TITLE
Add an 'Hello World' example to Stylable homepage

### PR DESCRIPTION
**Pull request by tomerlichtash**
Hey y'all,

Please review the following changes:

* Replaced top page logo+text image in with HTML Logo, Title and Subtitle.
* Added a new 'hello world' example to expose the essence of Stylable.
* Added a new section in the homepage for Code Completion and its animated gif.
* Moved the quote to the bottom of the page

@arnonkehat, @brucelawson: I'm waiting for updates from you regarding the final content of the TITLE tags, and for other comments as well.

### CODE REVIEW

**Comment by arnonkehat** [comment]
Texts for tooltips (@brucelawson, @idoros  and @AnnieLR please look at these as well):

-st-states: "We can target logical states of the component by declaring them on the root of the component"

:import : "Stylable allows you to import component interfaces and match their inner parts and states"

-st-default: someComponent: "'someComponent' becomes a type with a declared API"

SomeComponent::item: "Stylable allows you access to the internals of the component with a clean, safe API and possible code-completions"

SomeComponent:loading::item::before: "Stylable allows chaining selectors, so you can access the '::before' of an item, while the parent component is loading"

**Comment by arnonkehat** [comment]
Text for code completion section:
# Tooling
[your image]

Treating CSS as a type system opens up a whole new world of tooling. We can get code completion that hints at the internal structure of components, we can get types for our parameters, and more.  Our aim is to use Stylable to enhance to enhance the styling development process. 

To get the full Stylable experience, please install our VSCode Code Completion Plugin which suggests Stylable syntax, types and more.

**Comment by brucelawson** [comment]
Questions arising from looking more deeply at the code that (presumably) Jekyll gives us:

- why do we have <div class="main content"> rather than <main> ?
- why do we have pseudo-links like <a href="{{ site.baseurl }}/"><span class="logo"></span</a> with no link text (so nothing if someone's using assistive technology as no alt text for an image, either?). What's wrong with <a>home</a> ?

**Comment by idoros** [comment]
@bruce - this is just html templating. it can be easily fixed.

File: site/index.md [hunk 1]
```diff
@@ -4,42 +4,113 @@
...
-
-
-
-
-At Wix, we agree. We &hearts; CSS. Its simple, declarative syntax that is native in browsers is easily the fastest way to add styles to web pages and web apps. But when writing CSS that is scoped to individual components, developers have to maintain highly-specific selectors, using elaborate conventions to fake namespacing. Writing and maintaining CSS across large teams and large projects can be tricky.
+<span class="site-logo">![Stylable Intelligence](./branding/logo/PNG/96-logo-OnlySymbol.png)</span>
+<h1 class="site-title">Stylable</h1>
+<span class="site-description">is CSS for components</span>
+
+<div class="hello-world">
```
Comment (idoros) [inline site/index.md]: are we writing code markup directly to add the [title] attribute?

**Comment by idoros** [comment]
**-st-states:**  
`We can target logical states of the component by declaring them on the root of the component`
not just the root

-----

**:import :**  
`Stylable allows you to import component interfaces and match their inner parts and states`
also shared definitions like classes, variables, CSS / JS mixins and JS formatters

-----

**-st-default: someComponent**  
`'someComponent' becomes a type with a declared API`
import the style API of 'someComponent' to be used locally

-----

**SomeComponent::item**  
`Stylable allows you access to the internals of the component with a clean, safe API and possible code-completions`
Stylable exposes an clean, safe style API with code completions and validations

-----

**SomeComponent:loading::item::before**  
`Stylable allows chaining selectors, so you can access the '::before' of an item, while the parent component is loading`
Stylable allows chaining selectors, so you can target the `::before` of the `::item` element, while the parent component is `:loading`

**Comment by tomerlichtash** [comment]
@brucelawson, @arnonkehat, @idoros: Here's an editing suggestion, please review:

**-st-states:**  
`We can target logical states of the component by declaring them on the component's stylesheet`

-----

**:import :**  
`Stylable allows you to import component interfaces and match their inner parts,  states and shared definitions like classes, variables, JS/CSS mixins and JS formatters`

-----

**-st-default: SomeComponent**  
`we import 'someComponent' to be used locally as a style type with a declared API`

-----

**SomeComponent::item**  
`Stylable exposes an clean, safe style API with code completions and validations`

-----

**SomeComponent:loading::item::before**  
`Stylable allows chaining selectors, so you can access the '::before' of them `::item` element, while the parent component is loading`

**Comment by tomerlichtash** [comment]
Fixed few typos; ready for merge. Pending @NirBenita's ux/design comments.

**Pull request merged**